script/Arcueid/Day 7/07_00_ARC07_3.txt

```diff
@@ -269,17 +293,24 @@ Akiha pierces me with her unforgiving gaze.
 [sha:7e0792412cd7088b1d8efdbf9b8e318e381921b7]{
 -- Page 191, Offset 17039.
 -- ええい、こうなっては何を言っても無駄な気がするけど、誰も傷つかない……特に俺の平穏が脅かされない……弁明を繰り出さなければいけない。
-Fine. There's probably nothing I can say at this point, but if I don't want to hurt anyone... and especially if I don't want my peaceful life to be threatened... then I must come up with an excuse no matter what.
+Fine.
+ We're probably past the point where anything I say can make a difference,
+ but if I want to make sure nobody gets hurt...
+ and, of course, that my own peace isn't threatened...
+ I've no choice but to come up with some kind of explanation.
 }
 [sha:fac9223df3e7525ff58923d705a1db8fffa8ab04]{
 -- Page 192, Offset 17040.
 -- いけないんだけど―――そもそもの話、あそこで我関せずとニコニコ笑ってやがる吸血鬼女は、どんな<名目|めいもく>でうちにあがりこんだのだろう？
-I say that, but―――For starters, that vampire with a stupid grin on her face like it's none of her business. What sort of pretext did she use to get inside the mansion in first place?
+I've got to come up with an explanation, but before I get to that―――
+ Under what pretense this vampire woman,
```

Comment:
> under what pretense **did** this vampire,

I'd drop "woman" because that's just such a japanese thing, we don't really do that in english.